src/applicationContext.ts

```diff
@@ -12,19 +12,24 @@ import {LearningCatalogue} from './learning-catalogue'
 import {EnvValue} from 'ts-json-properties'
 import {CourseController} from './controllers/courseController'
 import {CourseFactory} from './learning-catalogue/model/factory/courseFactory'
-import {LearningProviderController} from './controllers/learningProvider/learningProviderController'
+import {LearningProviderController} from './controllers/LearningProvider/learningProviderController'
```

Comment:
These should be with lowercase 'L', you wont get a compile issue on a mac because its case insensitive, but when you run on azure, it'll claim that it the file cant be found